openshift-ci/capz-test-env.sh

```
@@ -80,6 +80,17 @@ else
   echo "$NAME_PREFIX" > "$NAME_PREFIX_FILE"
 fi
 
+# Randomize RESOURCEGROUPNAME per run to prevent parallel jobs from
+# interfering with each other's Azure resource groups.
+# Same SHARED_DIR pattern as NAME_PREFIX above.
+RESOURCEGROUPNAME_FILE="${SHARED_DIR:-/tmp}/resource-group-name"
+if [[ -f "$RESOURCEGROUPNAME_FILE" ]]; then
+  export RESOURCEGROUPNAME=$(cat "$RESOURCEGROUPNAME_FILE")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)